manifests/generated/virt-handler.yaml.in

```diff
@@ -60,7 +60,7 @@ spec:
         - mountPath: /var/lib/kubelet/device-plugins
           name: device-plugin
       hostPID: true
-      serviceAccountName: kubevirt-privileged
+      serviceAccountName: kubevirt-handler
```

Comment:
same here just rename to `virt-handler`

Reply:
I was attempting to be consistent with the kubevirt-apiserver and kubevirt-controller service accounts. I'd prefer not to unnecessarily change those two in this pull to maintain consistency with a 'virt-handler' rename, but I'll do what the community wants.

Reply:
yeah, the way you have it now is consistent with our other service accounts. for the sake of this PR I think we can continue with kubevirt-handler as the SA's name.